charts/civil-service/values.yaml

```diff
@@ -70,3 +70,5 @@ java:
         - robotics-notification-sender
         - robotics-notification-recipient
         - launch-darkly-sdk-key
+        - robotics-notification-multipartyrecipient
```

Comment:
this needs to get added to values.aat.template.yaml and values.preview.template.yaml files as well for respective environments

Reply:
mentioned yml files are updated